framework/src/wifi_manager/wifi_semaphore.h

```diff
@@ -0,0 +1,105 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef WIFI_SEMAPHORE_H
+#define WIFI_SEMAPHORE_H
+
+#include <semaphore.h>
+#include "wifi_common.h"
+
+/**
+ * @file wifi_semaphore.h
+ * @brief semaphore API
+ */
+
+/**
+ * @defgroup semaphore semaphore
```

Comment:
Those APIs are related with wifi semaphore.
but if you define this group to "semaphore", then category name in doxygen is set to semaphore.
that makes some confusion to users. isn't it?

Reply:
This is not public APIs. Anyway, change to Wi-Fi Manager group.